index.html

```diff
@@ -10,19 +10,16 @@
   <body>
     <section class="wrapper">
       <h1>Awesome Books</h1>
-      <div class="books-list"></div>
-      <div>
-        <label for="title">Title</label>
-        <input class="titleTxt" type="text" id="title" />
+      <div class="books-list">
+        
       </div>
-      <div>
-        <label for="Author">Author</label>
-        <input class="authorTxt" type="text" id="Author" />
-      </div>
-      <div>
-        <button type="submit" class="AddToShelfBtn">Add</button>
+      <div class="data-form">
+        <h2>Add a new book</h2>
+          <input class="titleTxt" type="text" id="title" placeholder="Title" />
+          <input class="authorTxt" type="text" id="Author" placeholder="Author" />
+          <button type="submit" class="AddToShelfBtn">Add</button>
       </div>
     </section>
-    <script src="assets/scripts/index.js"></script>
+     <script src="assets/scripts/script.js"></script> 
```

Comment:
- [ ] [Optional] You refactored your app to use classes in the new file but I can see the old file `index.js` still exists so please remove any unused files or code. Other than that you have done a great job.